.reuse/dep5

```diff
@@ -25,11 +25,11 @@ Disclaimer: The code in this project may include calls to APIs (“API Calls”)
  parties the right to use of access any SAP External Product, through API Calls.
 
 Files: *
-Copyright:  2020-2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
+Copyright:  2020-2022 SAP SE or an SAP affiliate company and project "Sailor" contributors
 License: Apache-2.0
 
 Files: docs/_templates/apidoc/module.rst_t, docs/_templates/apidoc/package.rst_t
-Copyright:  2007-2021 by the Sphinx team. Modifications 2021 SAP SE or an SAP affiliate company and project "Sailor" contributors
+Copyright:  2007-2021 by the Sphinx team. Modifications 2022 SAP SE or an SAP affiliate company and project "Sailor" contributors
```

Comment:
sorry for being late to the reviewing party. I think this needs to stay also in 2021.